index.js

```diff
@@ -1,30 +1,51 @@
+const multiRet = ['push', 'pop', 'shift', 'unshift']
```

Comment:
Completely respect that, will change if you say you are sure. But we gained 10 bytes in the minified output when I replaced all `const`s with `let`s. Are you sure you want this?